.github/workflows/build_linux_packages.yml

```diff
@@ -59,8 +79,65 @@ jobs:
           restore-keys: |
             linux-build-packages-manylinux-v2-
 
+      - name: Fetch submodule from remote
+        run: |
+          proj_name="${{ inputs.project_name }}"
+          proj_branch="${{ inputs.project_branch }}"
+          proj_seturl="${{ inputs.project_seturl }}"
```

Comment:
Let's move all this bash to a Python script so we can

1. Reuse it between workflows
2. Run it locally
3. Add tests for it
4. Add more sophisticated logging and error handling